commons/src/main/java/org/infinispan/commons/io/ExposedByteArrayOutputStream.java

```diff
@@ -20,7 +18,7 @@
  * @since 4.0
  */
 @NotThreadSafe
-public final class ExposedByteArrayOutputStream extends ByteArrayOutputStream implements ByteOutput {
```

Comment:
It's because `ByteOutput` is simply:

```java
public interface ByteOutput extends Closeable, Flushable {
   void write(int var1) throws IOException;

   void write(byte[] var1) throws IOException;

   void write(byte[] var1, int var2, int var3) throws IOException;
}
```

Reply:
Then why do we remove the interface?

Reply:
Because it's from jboss-marshalling :smile:

Reply:
`org.jboss.marshalling.ByteOutput`

Reply:
Ah 👍 For some reason I thought it was a standard java class, haha.